data/schemas/currency/code.json

```diff
@@ -38,12 +38,12 @@
           "title": "Macanese Pataca"
         },
         {
-          "const": "HTG",
-          "title": "Haitian Gourde"
+          "const": "BWP",
```

Comment:
I'd try and avoid these types of random generate changes if possible. Would be good to understand why this is happening.